docs/release-notes.rst

```diff
@@ -25,6 +25,9 @@ Features
 * Fix wrong title on Edit page (`#578
   <https://github.com/release-monitoring/anitya/pull/578>`_).
 
+* Default custom regex is now configurable (`#571
+  <https://github.com/release-monitoring/anitya/pull/571>`_).
```

Comment:
The github and gitlab backend changes are also in this release, right?

Reply:
Yes, they are mentioned in this PR https://github.com/release-monitoring/anitya/pull/601
So we should probably merge https://github.com/release-monitoring/anitya/pull/601 first.
My bad.